MathPortal/MathPortal/Source/Tools/CustomTableView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CustomTableView.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 25/08/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+protocol CustomTableViewDelegate: class {
+    func customTableView(sender: CustomTableView, didRemoveCellAt previousIndexPath: IndexPath, didInsertCellAt currentIndexPath: IndexPath)
```

Comment:
This looks more like `func customTableView(_ sender: CustomTableView, didMoveCellAt fromIndexPath: IndexPath, to toIndexPath: IndexPath)`